package.json

```diff
@@ -26,5 +27,6 @@
     "npm": ">=7.0.0",
     "node": ">=14.0.0"
   },
-  "packageManager": "npm@8.1.2"
+  "packageManager": "npm@8.1.2",
+  "prettier": {}
```

Comment:
The only change you really need to care about. The rest is automatic.

I also kept the `format` npm script for future migrations.